dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -920,110 +841,115 @@ void CommandBarFlyoutCommandBar::OnKeyDown(
         const bool isDown = args.Key() == winrt::VirtualKey::Down;
         const bool isUp = args.Key() == winrt::VirtualKey::Up;
 
-        auto moreButton = m_moreButton.get();
+        // The primary commands and the more button are the only controls accessible
+        // using left and right arrow keys. All of the commands are accessible using
+        // the up and down arrow keys.
+        auto horizontallyAccessibleControls = winrt::make<Vector<winrt::Control>>();
+        auto verticallyAccessibleControls = winrt::make<Vector<winrt::Control>>();
```

Comment:
Do you think it would be possible/reasonable to do this just once anytime the commands change, instead of every time an arrow key is pressed? It seems like a lot, but maybe it's not really that expensive -- just a thought.

Reply:
I had that same thought, actually.  I can probably have it populate these lists when the primary or secondary command vectors change rather than every single time we do this.